src/main/java/nl/tudelft/contextproject/script/Script.java

```diff
@@ -123,6 +135,13 @@ private void initPresetLoading() {
             t.initPreset();
         }
     }
+    
+    /**
+     * Init the timer for calling updateOldCam() with a 1 second delay.
+     */
+    private void initTimer() {
```

Comment:
There used to be more in the method, but now that there isn't, I removed it as you suggested.